R/source.R

```diff
@@ -437,10 +443,25 @@ generate_cargo.toml <- function(libname = "rextendr",
       dependencies
     ),
     `patch.crates-io` = patch.crates_io,
-    features = features
+    features = features,
+    `profile.optim` = list(
+      inherits = "release",
+      lto = "thin",
+      `opt-level` = 3,
+      panic = "abort",
+      `codegen-units` = 1
+    )
   )
 }
 
+generate_cargo_config.toml <- function() {
+  to_toml(
+    build = list(
+      rustflags = array(c("-C", "target-cpu=native")),
```

Comment:
```suggestion
      rustflags = c("-C", "target-cpu=native"),
```

`array()` is only needed to distinguish scalar values from vectors of length `1` (by adding `dim` attribute).